crates/rover-client/src/error.rs

```diff
@@ -83,6 +83,11 @@ pub enum RoverClientError {
     #[error("Could not find graph with name \"{graph}\"")]
     NoService { graph: String },
 
+    /// if someone attempts to get a core schema from a supergraph that has
+    /// no composition results we return this error.
+    #[error("There has been no successful composition for supergraph \"{graph}\".")]
```

Comment:
Nit: we use either present or past tense in our error messages. This would be the only time we would you present perfect. How about this instead?
```suggestion
    #[error("There is no successful composition for supergraph \"{graph}\".")]
```